vividus-plugin-xml/src/main/java/org/vividus/steps/xml/XmlSteps.java

```diff
@@ -167,18 +181,21 @@ public void validateXmlAgainstXsd(String xml, String xsd)
      * Validates if the XML document is well formed
      *
      * @param xml The XML document
+     * @return whether the XML document if well formed
      */
     @Then("XML `$xml` is well formed")
-    public void validateXmlIsWellFormed(String xml)
+    public boolean isXmlWellFormed(String xml)
     {
         try
         {
             XmlUtils.convertToDocument(xml);
             softAssert.recordPassedAssertion("The XML document is well formed");
```

Comment:
No reason to record the passed assertions for other steps except the current one